File: petstore/java/src/main/java/oseg/petstore_examples/UpdateUserExample.java
package oseg.petstore_examples;

import org.openapitools.client.ApiException;
import org.openapitools.client.Configuration;
import org.openapitools.client.api.*;
import org.openapitools.client.auth.*;
import org.openapitools.client.JSON;
import org.openapitools.client.model.*;

import java.io.File;
import java.time.LocalDate;
import java.time.OffsetDateTime;
import java.util.ArrayList;
import java.util.List;
import java.util.Map;

public class UpdateUserExample
{
    public static void main(String[] args)
    {
        var config = Configuration.getDefaultApiClient();
        config.setApiKey("YOUR_API_KEY");

        var user = new User();
        user.id(12345L);
        user.username("new-username");
        user.firstName("Joe");
        user.lastName("Broke");
        user.email("some-email@example.com");
        user.password("so secure omg");
        user.phone("555-867-5309");
        user.userStatus(1);

        try
        {
            new UserApi(config).updateUser(
                "my-username", // username
                user
            );
        } catch (ApiException e) {
            System.err.println("Exception when calling User#updateUser");
            System.err.println("Status code: " + e.getCode());
            System.err.println("Reason: " + e.getResponseBody());
            System.err.println("Response headers: " + e.getResponseHeaders());
            e.printStackTrace();
        }
    }
}

File: petstore/kotlin/src/main/kotlin/oseg/petstore_examples/UpdateUserExample.kt
package oseg.petstore_examples

import org.openapitools.client.infrastructure.*
import org.openapitools.client.apis.*
import org.openapitools.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map

class UpdateUserExample
{
    fun updateUser()
    {
        ApiClient.apiKey["api_key"] = "YOUR_API_KEY"

        val user = User(
            id = 12345,
            username = "new-username",
            firstName = "Joe",
            lastName = "Broke",
            email = "some-email@example.com",
            password = "so secure omg",
            phone = "555-867-5309",
            userStatus = 1,
        )

        try
        {
            UserApi().updateUser(
                username = "my-username",
                user = user,
            )
        } catch (e: ClientException) {
            println("4xx response calling UserApi#updateUser")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling UserApi#updateUser")
            e.printStackTrace()
        }
    }
}
